doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -134,16 +148,81 @@ pass testing for ``start_test -performance``).
 
 .. _Creating a Simple Test: https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst#creating-a-simple-test
 
+.. _Design
+
+Design
+------
+
+.. _Creating a design issue
+
+Creating a design issue
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The design issue should aim to clearly and concisely present some design
+decisions. Here are some strategies to make a good design issue:
+
+- Summarize issue at the top, preferably include concise code example(s).
+- Try not to make initial issue too long in order to lower the barrier for
+  someone getting involved.
+- Clearly define the problem this proposal aims to solve.
+- It can be useful to consider alternative solutions to the problem and
+  describe the trade-offs among them.
+- A design issue should have two developers not involved in the effort to review.
+- To find reviewers, send an email to chapel-developers_.
+
+  * Use a prefix of ``[Design]`` in the subject header.
+  * Include a short summary of the topic, potentially including motivation and/or an example.
+  * Include a link to your issue.
+  * Ask for people to volunteer to be involved in design discussions.
```

Comment:
Should we be more explicit here saying that this email shouldn't be a copy/paste of the issue? This is already said by "short summary" in the second bullet, but maybe we can repeat it so that we don't get pages-long emails about design issues

Reply:
Probably doesn't hurt. Good idea.